pkg/consuldp/bootstrap.go

```diff
@@ -27,9 +27,8 @@ const (
 )
 
 // bootstrapConfig generates the Envoy bootstrap config in JSON format.
-func (cdp *ConsulDataplane) bootstrapConfig(ctx context.Context) (*bootstrap.BootstrapConfig, []byte, error) {
+func (cdp *ConsulDataplane) getBootstrapParams(ctx context.Context) (*pbdataplane.GetEnvoyBootstrapParamsResponse, error) {
```

Comment:
dns proxy requires bootstrap params to set the namespace and partition.....but it does not require the bootstrap config itself.  So we are splitting this into two functions, so that `startDNSProxy()` can get require bootstrap params in its signature rather than just assuming it is already set on the struct.